inc/saiqueue.h

```diff
@@ -318,6 +318,16 @@ typedef struct _sai_queue_deadlock_notification_data_t
     /** Deadlock event */
     sai_queue_pfc_deadlock_event_type_t event;
 
+    /**
+     * @brief Application based recovery management indicator.
+     *
+     * This is a return value from host adapter.
+     * If set to TRUE then host application will manage deadlock recovery
+     * else SAI adapter or SDK will manage deadlock recovery
+     * and also generate recovery ended notification.
+     */
```

Comment:
should we also add a comment that this attribute is only APPLICABLE when event == SAI_QUEUE_PFC_DEADLOCK_EVENT_TYPE_DETECTED?